ddtrace/contrib/pylons/middleware.py

```diff
@@ -47,6 +51,20 @@ def _distributed_tracing(self):
     def _distributed_tracing(self, distributed_tracing):
         ddconfig.pylons["distributed_tracing"] = asbool(distributed_tracing)
 
+    def _unlist_multidict_params(self, multidict):  # type: (webob.multidict.MultiDict) -> Dict[str, Any]
+        # Pylons multidict always return values in a list. This will convert lists with a
+        # single value
+        # into the un-listed value
+        new_dict = {}  # type: Dict[str, Any]
+
+        for k, v in six.iteritems(multidict):
+            if isinstance(v, typing.List) and len(v) == 1:
```

Comment:
nit: `isinstance(v, list)` so you can avoid ` import typing`

Reply:
Done, thanks!